src/app/page/card/card.component.ts

```diff
@@ -14,12 +14,13 @@ export class CardComponent implements OnInit {
   public static deckId: number;
   public routing = false;
   public questionNumber = 0;
-  public MAX_QUANTITY_CARD = ORLPSettings.MAX_QUANTITY_CARD;
+  public MAX_QUANTITY_CARD;
```

Comment:
Sonar: Local variable and method parameter names should comply with a naming convention.